events/_event-post-macros.html

```diff
@@ -104,23 +115,55 @@
 
    Create HTML event meta address markup when given:
 
-   post: A post date from a query result.
+   post                     : A post date from a query result.
+
+   address_format (optional): A python string format to display to 
+                              address information  
+                              (e.g. '{venue} - {city}, {state}').
 
    ========================================================================== #}
 
-{% macro event_meta_address(post) %}
-    {%- set city = (post.location or 'Washington DC') -%}
-    {%- set state = (post.state or '') -%}
-    {%- set venue = (post.venue or 'CFPB') -%}
+{% macro event_meta_address(post, address_format='%city %state') %}
```

Comment:
Just checking that default value shouldn't be `address_format='{city} {state}' like https://github.com/cfpb/cfgov-refresh/pull/316/files#diff-6bf9522468f4795c50b2040f06dbc410R100